qase-playwright/src/reporter.ts

```diff
@@ -182,17 +269,49 @@ export class PlaywrightQaseReporter implements Reporter {
       relations: [],
       run_id: null,
       signature: '',
-      steps: PlaywrightQaseReporter.transformSteps(result.steps, null),
+      steps: this.transformSteps(result.steps, null),
       // suiteTitle: PlaywrightQaseReporter.transformSuiteTitle(test),
-      testops_id: ids[0] ?? null,
-      title: test.title,
-    });
+      testops_id: null,
+      title: testCaseMetadata.title === '' ? test.title : testCaseMetadata.title,
+    };
+
+    let ids: number[];
+    if (testCaseMetadata.ids.length > 0) {
+      ids = testCaseMetadata.ids;
+    } else {
+      ids = PlaywrightQaseReporter.qaseIds.get(test.title) ?? [];
+    }
+
+    if (ids.length == 0) {
+      this.reporter.addTestResult(testResult);
+      return;
+    }
+
+    // if we have multiple ids, we need to create multiple test results and set duration to 0 for all but the first one
```

Comment:
why not the same duration for all of them?

Reply:
Because we ran the test once and added the result several times. For example, the test ran for 5 seconds. If the test has more than one ID specified, then in the statistics the total test execution time will be increased by 5 * the number of IDs. But the test was only run once.